alert-database/src/main/resources/liquibase/6.13.0/concrete-saml-configuration-table.xml

```diff
@@ -21,16 +21,16 @@
                 <constraints nullable="true"/>
             </column>
             <column name="enabled" type="BOOLEAN" defaultValueBoolean="false">
-                <constraints nullable="false"/>
```

Comment:
Why are all these fields now nullable?  If they are required fields for saml then the field shouldn't be nullable.

Reply:
These are the fields we discussed are required. 

Metadata url OR file path required
entityBaseUrl
entityid

I previously had everything set to false so corrected the non-required ones to false here

Reply:
Enabled is required as well in the context of the database.  That cannot be null.

Reply:
Changed enabled, force_auth, and want_assertions_signed to nullable false